content/docs/command-reference/run.md

```diff
@@ -270,15 +270,19 @@ $ dvc run -n second_stage './another_script.sh $MYENVVAR'
   affect any DVC operations.
 
 - `--live <path>` - specify the directory `path` for
-  [Dvclive](/doc/dvclive/dvclive-with-dvc) to write logs in. `path` will be
-  tracked (<abbr>cached</abbr>) by DVC. Saved in the `live` field of `dvc.yaml`.
+  [DVCLive](/doc/dvclive/user-guide/dvclive-with-dvc) to write logs in. `path`
```

Comment:
BTW (unrelated) did we ever setup redirects for these link changes? Check /redirects-list.json

Reply:
Didn't know about that, thanks!

Reply:
You are refering to https://github.com/iterative/dvc.org/blob/master/content/linked-terms.js , right?

Reply:
https://github.com/iterative/dvc.org/blob/master/redirects-list.json

Reply:
We use it when docs have been published earlier. Doing a redirect helps search engines transfer ranking to the new URL. Also if anyone linked the old URL from their blogs or something, this way their links won't brake.

Reply:
Oh, got it!